src/DetailsView/components/issue-filing-dialog.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Dialog, DialogFooter, DialogType } from 'office-ui-fabric-react/lib/Dialog';
+import * as React from 'react';
+
+import { BugFilingService } from '../../bug-filing/types/bug-filing-service';
+import { CreateIssueDetailsTextData } from '../../common/types/create-issue-details-text-data';
+import { BugServiceProperties } from '../../common/types/store-data/user-configuration-store';
+import { ActionAndCancelButtonsComponent } from './action-and-cancel-buttons-component';
+import { NamedSFC } from '../../common/react/named-sfc';
+
+export interface IssueFilingDialogProps {
+    deps: IssueFilingDialogDeps;
+    isOpen: boolean;
+    selectedBugFilingService: BugFilingService;
+    selectedBugData: CreateIssueDetailsTextData;
+    selectedBugFilingServiceData: BugServiceProperties;
+    bugFileTelemetryCallback: (ev: React.SyntheticEvent) => void;
+    onClose: (ev: React.SyntheticEvent) => void;
+}
+
+export interface IssueFilingDialogDeps {}
+
+const titleLabel = 'Specify issue filing location';
+
+export const IssueFilingDialog = NamedSFC<IssueFilingDialogProps>('IssueFilingDialog', props => {
+    const { selectedBugFilingService, selectedBugFilingServiceData, selectedBugData, bugFileTelemetryCallback, onClose, isOpen } = props;
+
+    return (
+        <Dialog
+            className={'issue-filing-dialog'}
+            hidden={!isOpen}
+            dialogContentProps={{
+                type: DialogType.normal,
+                title: titleLabel,
+                titleId: 'issue-filing-dialog-title',
+                subText: 'This configuration can be changed again in settings.',
+                subTextId: 'issue-filing-dialog-subtext',
+                showCloseButton: false,
+            }}
+            modalProps={{
+                isBlocking: false,
+                containerClassName: 'insights-dialog-main-override',
+            }}
+            onDismiss={onClose}
+        >
+            <DialogFooter>
+                <ActionAndCancelButtonsComponent
+                    isHidden={false}
+                    primaryButtonDisabled={selectedBugFilingService.isSettingsValid(selectedBugFilingServiceData)}
+                    primaryButtonOnClick={bugFileTelemetryCallback}
+                    cancelButtonOnClick={onClose}
+                    primaryButtonHref={selectedBugFilingService.createBugFilingUrl(selectedBugFilingServiceData, selectedBugData)}
+                    primaryButtonText={'File issue'}
```

Comment:
I guess we may want to get this string (`'File issue'`) from the service but I'm fine if we do that on a different PR

Reply:
This isn't coming from the service since we aren't don't that as a part of this feature as of yet.